doc/contributors.rst

```diff
@@ -31,4 +31,4 @@ dramatically.
 
 Michael R. Crusoe took over maintainership June, 2013.
 
-MRC 2015-07-31
+Last updated by MRC on 2015-07-31
```

Comment:
Did we want to expand this section to include all other contributors? I assume not, but it might be worth distinguishing between contributours  here and authors elsewhere